docs/modernization/MediaPlayerElement.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
@normesta / @mcleanbyron Could you review this new documentation for Media Player Element?